### ISSUE TITLE
[docs] each augur command should have its own page

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/augur#306]
Currently all CLI commands are on the same page: https://nextstrain-augur.readthedocs.io/en/stable/cli.html. In a docs meeting last week we decided that these should all have their own pages. "/cli.html" should remain, perhaps providing an overview of all available commands.

**Comment by kairstenfay** [comment]
@jameshadfield Do you want to replace the `/cli.html` page with `/augur.html` ([from the docs page](https://nextstrain-augur.readthedocs.io/en/stable/augur.html)?

**Comment by jameshadfield** [comment]
> Can you explain to me the need for a /cli page in addition to an /api page?

Not really. I would prefer the simplest solution. (Redirects will be needed for pages we get rid of.) Something like:
```
- Installation
- Augur commands -> overview page
    - parse
    - filter
    - ...
- Examples of augur
- Development (API)
- Authors
```

**Comment by kairstenfay** [comment]
Can you please explain the difference between what you want displayed under `Augur commands` vs. `Development (API)`? How do the contents of the `parse` vs. `augur.parse` pages differ?

**Comment by jameshadfield** [comment]
> the difference between what you want displayed under Augur commands vs. Development (API)?

The first is for those who want to use the command, e.g. `augur filter` and shall contain a human-written section explaining the command, with links to tutorials etc (this can be blank to start with). It'll also have an automatically generated listing of the help, e.g. the same text as `augur filter --help`.

The second is all the internal function docstrings etc. For instance https://nextstrain-augur.readthedocs.io/en/stable/augur.tree.html. This is not intended for people simply wishing to run augur from the command line.

**Comment by kairstenfay** [comment]
Thank you, that's very helpful.

I'm going to create a bunch of new files for augur commands (parse, filter, etc.) using `argparse` to create page stubs. Then that will allow people to come in and add example code to those pages later.

The API pages are already generated programmatically, as you stated, so I'll mostly leave those be.

**Comment by kairstenfay** [comment]
Closed via #359
